### PR TITLE
[14.0][IMP] document_page: copy document function.

### PR DESCRIPTION
**Pull request by angelgarciadelachica**
The copy function is modified to create a copy named original_name+(copy) and with the same content

### CODE REVIEW

**Comment by angelgarciadelachica** [comment]
@ValentinVinagre @HaraldPanten

**Comment by ValentinVinagre** [comment]
@angelgarciadelachica angel adds a small test that checks the new functionality please.

File: document_page/models/document_page.py [hunk 1]
```diff
@@ -182,3 +182,14 @@ def unlink(self):
...
+
+    def copy(self, default=None):
+        default = dict(
+            default or {},
+            name=_("%s (copy)") % self.name,
+            content=self.content,
+            draft_name="1.0",
+            draft_summary="summary",
+        )
+        result = super(DocumentPage, self).copy(default=default)
```
Comment (ValentinVinagre) [inline document_page/models/document_page.py]: you can return directly the result

File: document_page/models/document_page.py [hunk 2]
```diff
@@ -182,3 +182,14 @@ def unlink(self):
...
         menus.unlink()
         return res
+
+    def copy(self, default=None):
+        default = dict(
+            default or {},
+            name=_("%s (copy)") % self.name,
+            content=self.content,
+            draft_name="1.0",
+            draft_summary="summary",
```
Comment (ValentinVinagre) [inline document_page/models/document_page.py]: this "text" need to be translatable with the _()

**Comment by angelgarciadelachica** [comment]
@ValentinVinagre I think I have entered all your suggestions. Anything else?

File: document_page/models/document_page.py [hunk 3]
```diff
@@ -182,3 +182,13 @@ def unlink(self):
         res = super().unlink()
         menus.unlink()
         return res
+
+    def copy(self, default=None):
+        default = dict(
+            default or {},
+            name=_("%s (copy)") % self.name,
+            content=self.content,
+            draft_name=_("1.0"),
```
Comment (ValentinVinagre) [inline document_page/models/document_page.py]: this should not be translatable

**Review by ValentinVinagre — APPROVED** [review]
LGTM 👍🏻

**Review by HaraldPanten — APPROVED** [review]
Functional review in runboat.

**Pull request merged**